index.bs

```diff
@@ -358,324 +493,120 @@ For the main <dfn>sanitize</dfn> operation, using a {{ParentNode}} |node|, a
-   1. If |config|["{{SanitizerConfig/removeElements}}"] [=map/exists=]:
-       1. Set |result|["{{SanitizerConfig/elements}}"] to the
-          [=set/difference=] of |default|["{{SanitizerConfig/elements}}"]
-          and |result|["{{SanitizerConfig/removeElements}}"].
-       1. [=set/Remove=] "{{SanitizerConfig/removeElements}}" from |result|.
-   1. If neither |config|["{{SanitizerConfig/elements}}"] nor
-      |config|["{{SanitizerConfig/removeElements}}"] [=map/exist=]:
-      1. Set |result|["{{SanitizerConfig/elements}}"] to
-         |default|["{{SanitizerConfig/elements}}"].
-   1. If |config|["{{SanitizerConfig/attributes}}"] [=map/exists=]:
-      1. Let |attributeBlockList| be the [=set/difference=] between
-         [=known attributes=] and |default|["{{SanitizerConfig/attributes}}"];
-      1. Set |result|["{{SanitizerConfig/attributes}}"] to the
-         [=set/difference=] of |result|["{{SanitizerConfig/attributes}}"] and
-         |attributeBlockList|.
-   1. If |config|["{{SanitizerConfig/removeAttributes}}"] [=map/exists=]:
-       1. Set |result|["{{SanitizerConfig/attributes}}"] to the
-          [=set/difference=] of |default|["{{SanitizerConfig/attributes}}"]
-          and |result|["{{SanitizerConfig/removeAttributes}}"].
-       1. [=set/Remove=] "{{SanitizerConfig/removeAttributes}}" from |result|.
-   1. If neither |config|["{{SanitizerConfig/attributes}}"] nor
-      |config|["{{SanitizerConfig/removeAttributes}}"] [=map/exist=]:
-      1. Set |result|["{{SanitizerConfig/attributes}}"] to
-         |default|["{{SanitizerConfig/attributes}}"].
-1. Else (if not |safe|):
-   1. If neither  |config|["{{SanitizerConfig/elements}}"] nor
-      |config|["{{SanitizerConfig/removeElements}}"] [=map/exist=]:
-      1. Set |result|["{{SanitizerConfig/elements}}"] to
-         |default|["{{SanitizerConfig/elements}}"].
-   1. If neither  |config|["{{SanitizerConfig/attributes}}"] nor
-      |config|["{{SanitizerConfig/removeAttributes}}"] [=map/exist=]:
-      1. Set |result|["{{SanitizerConfig/attributes}}"] to
-         |default|["{{SanitizerConfig/attributes}}"].
-1. [=Assert=]: |result| is [=SanitizerConfig/valid=].
-1. [=Assert=]: |result| is [=SanitizerConfig/canonical=].
+Note: While this algorithm is called [=remove unsafe=], we use
+    <a href="#security-considerations">the term "unsafe" strictly in the sense
+    of this spec</a>, to denote content that will
+    execute JavaScript when inserted into the document. In other words, this
+    method will remove oportunities for XSS.
+
+To <dfn for="SanitizerConfig">remove unsafe</dfn> from a |config|, do this:
+
+1. [=Assert=]: The [=built-in safe baseline config=] has
+   {{SanitizerConfig/removeElements}} and {{SanitizerConfig/removeAttributes}}
+   keys set, but not {{SanitizerConfig/elements}},
+   {{SanitizerConfig/replaceWithChildrenElements}}, or
+   {{SanitizerConfig/attributes}}.
+1. Let |result| be a copy of |config|.
+1. [=list/For each=] |elem| in
```

Comment:
Sometimes you use elem, sometimes, item, and sometimes element. I would settle on element for all of these. I'd generally prefer less abbreviations in prose, so also attribute and configuration instead of attr and config.

Reply:
Mostly done. I consistently use element, attribute, and configuration now. I still use item now and then, but only for contexts where it could be either an element or an attribute. Happy to rename that, but I couldn't really come up with a nice name for element_or_attribute.